src/config/dynamic-theme-fixes.config

```diff
@@ -1948,6 +1948,13 @@ INVERT
 
 ================================
 
+laptopmag.com
+
+INVERT
+div#publisherDetails.logo
+
```

Comment:
Add `.qc-cmp-publisher-logo` The cookie wall shows a logo as well.

Reply:
Didn't have that banner since i'm using ubo. :)

Reply:
uBlock Origin?

Reply:
Yes.